packages/adblocker-electron-preload/src/index.ts

```diff
@@ -70,8 +41,8 @@ if (window === window.top && window.location.href.startsWith('devtools://') ===
       () => {
         DOM_MONITOR = new DOMMonitor((update) => {
           if (update.type === 'features') {
-            getCosmeticsFiltersUpdate({
-              ...update,
+            injectCosmeticFilters({
+               ...update,
```

Comment:
```suggestion
              ...update,
```

Reply:
this single whitespace breaks the linter

Reply:
fixed it, thanks!